cmd/kyma/upgrade/cmd.go

```diff
@@ -33,20 +33,29 @@ func NewCmd(o *Options) *cobra.Command {
 
 	cobraCmd := &cobra.Command{
 		Use:   "upgrade",
-		Short: "Upgrades Kyma to match  the CLI version.",
-		Long:  `Use this command to upgrade the Kyma version on a cluster so that it matches the CLI version`,
+		Short: "Upgrades Kyma",
+		Long:  `Use this command to upgrade the Kyma version on a cluster. If no source is specified, then Kyma is upgraded to the same version as the CLI`,
 		RunE:  func(_ *cobra.Command, _ []string) error { return cmd.Run() },
 	}
 
 	cobraCmd.Flags().BoolVarP(&o.NoWait, "noWait", "n", false, "Determines if the command should wait for the Kyma upgrade to complete.")
 	cobraCmd.Flags().StringVarP(&o.Domain, "domain", "d", defaultDomain, "Domain used for the upgrade.")
 	cobraCmd.Flags().StringVarP(&o.TLSCert, "tlsCert", "", "", "TLS certificate for the domain used for the upgrade. The certificate must be a base64-encoded value.")
+	cobraCmd.Flags().StringVarP(&o.Source, "source", "s", "", `Upgrade source. 
+	- To use the specific release, write "kyma upgrade --source=1.3.0".
```

Comment:
should we call it target instead of source ? Usually in target you provide a target version one wants to upgrade to

Reply:
I think that `target` is a better name for the upgrade command, but then should it be `target-path` instead of `src-path`?

Reply:
@a-thaler what is your opinion here?
Should we have the flags named as `source` and `src-path` the same way as in the install command
or is it better to name them `target` and `target-path`?

Reply:
Let us leave it now consistent with the install command and if we got feedback that it is confusing, then we could change it